ui/app/components/app/asset-list-item/asset-list-item.js

```diff
@@ -13,11 +20,22 @@ const AssetListItem = ({
   iconClassName,
   onClick,
   tokenAddress,
+  tokenSymbol,
   tokenImage,
   warning,
   primary,
   secondary,
 }) => {
+  const t = useI18nContext()
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const sendTokenEvent = useMetricEvent({
+    eventOpts: {
+      category: 'Navigation',
+      action: 'Home',
```

Comment:
I'm sure that this event needs to be updated, as it's not being triggered from the home page anymore but rather the asset list. Unsure of what to change it to

Reply:
I think it's fine. It's still on the hope page technically - that's the only place the asset list is shown.